src/box/vinyl.c

```diff
@@ -1150,6 +1150,12 @@ vinyl_space_check_format(struct space *space, struct tuple_format *format)
 		if (++loops % VY_YIELD_LOOPS == 0)
 			fiber_sleep(0);
 		ERROR_INJECT_YIELD(ERRINJ_CHECK_FORMAT_DELAY);
+		ERROR_INJECT_COUNTDOWN(ERRINJ_CHECK_FORMAT_DELAY_COUNTDOWN, {
+			struct errinj *e =
+				errinj(ERRINJ_CHECK_FORMAT_DELAY, ERRINJ_BOOL);
+			e->bparam = true;
+			ERROR_INJECT_YIELD(ERRINJ_CHECK_FORMAT_DELAY);
+		});
```

Comment:
Same here. Or do I miss why it wouldn't work?

Reply:
`ERROR_INJECT_COUTNDOWN` executes the code only when it has the `0` value set. The default value is `-1`, so `ERRINJ_CHECK_FORMAT_DELAY` won’t work.

Or maybe I don’t understand what you mean by
> If you switch these 2 ERROR_INJECT calls

Reply:
I guess we will ignore this for now.

Reply:
Sorry, I forgot to push the fix.